### DISCUSSION TITLE
Could not get the stack frames of error: TypeError: Cannot read properties of null (reading 'length')

### DISCUSSION THREAD

**Discussion by 18211365467** [antvis/G2#5696] [Q&A]
### 问题描述

这个报错一般是因为什么

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by pearmini** [comment]
这个不清楚了，可以提供复现 demo
